tensorflow/lite/interpreter_test.cc

```diff
@@ -303,7 +303,7 @@ TEST(BasicInterpreter, CheckArenaAllocation) {
 
   std::vector<int> sizes{2048, 4096, 1023, 2047, 1021,
                          2047, 1023, 2046, 0,    2048};
-  for (int i = 0; i < sizes.size(); ++i) {
+  for (size_t i = 0; i < sizes.size(); ++i) {
     interpreter.SetTensorParametersReadWrite(i, kTfLiteUInt8, "", {sizes[i]},
```

Comment:
You'll need a static_cast in at least one place, otherwise revert this change.

Reply:
Sorry, to be specific, SetTensorParametersReadWrite takes an integer index, not a size_t.

Reply:
@jdduke , thanks for pointing that out, i have changed the code as per our suggestion i.e used static_cast. Kindly check and approve.